src/Microsoft.ML.StandardLearners/AssemblyInfo.cs

```diff
@@ -7,6 +7,7 @@
 
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.EntryPoints" + PublicKey.Value)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Legacy" + PublicKey.Value)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.LightGBM" + PublicKey.Value)]
```

Comment:
LightGBM uses Ova, but it's public. What is root cause of this?

Reply:
LightGBM uses `OvaModelParameters.Create()`, which I made internal. I want the API to be consistent with one public constructor, and remove all the public `Create` methods.